app/controllers/application_controller.rb

```diff
@@ -16,6 +16,7 @@ class ApplicationController < ActionController::Base
   private
 
   def after_sign_out_path_for(*)
-    '/Shibboleth.sso/Logout'
+    '/Shibboleth.sso/Logout' unless Rails.env.development?
```

Comment:
What path do we want when `RAILS_ENV=test`?

Reply:
I think you want an explicit `return` here too.

Reply:
Good point, should tests run against the same as dev? Or should shib be assumed for testing?

Reply:
I'm guessing we don't want to deal with shib in Travis, so maybe we want:

```ruby
return '/Shibboleth.sso/Logout' if Rails.env.production?
```

Reply:
It's working without an explicit return, but I think you are right, for clarity here.